ihatemoney/templates/layout.html

```diff
@@ -6,6 +6,11 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel=stylesheet type=text/css href="{{ url_for("static", filename='css/main.css') }}">
+    <!-- Load d3.js -->
+    <script src="https://d3js.org/d3.v6.min.js" charset="utf-8"></script>
```

Comment:
If possible, we shouldn't rely on external hosts to provide us with the javascript files. Could you please include it in this repository? Thanks!